pkg/config/config_template.yaml

```diff
@@ -505,6 +505,49 @@ api_key:
   #
   # validation_period: 60
 
+
+## @param fips - custom object - optional
+## Enter specific configurations for using the FIPS proxy.
+## Uncomment this parameter and the one below to enable them.
+#
```

Comment:
Do we want to add a `BETA` comment somewhere ?

Reply:
I would say yes imho

Reply:
We should for sure have a big note here that this feature requires the installation and configuration of the datadog-fips-proxy as well otherwise users might think that flipping this flag is the only thing they need to do.